cmd/ical-relay/templates/static/js/dnd.js

```diff
@@ -0,0 +1,50 @@
+const dropzone = document.getElementById("dropzone");
+
+document.addEventListener("dragenter", function(event) {
+    event.preventDefault();
+    dropzone.style.display = "flex";
+});
+
+document.addEventListener("dragover", function(event) {
+    event.preventDefault();
+});
+
+
+dropzone.addEventListener("drop", function(event) {
+    event.preventDefault();
+    dropzone.style.display = "none";
+});
+
+
+dropzone.addEventListener("drop", function(event) {
+    event.preventDefault();
+    const { files } = event.dataTransfer;
+    handleDroppedFiles(files);
+
+});
+
+async function handleDroppedFiles(files) {
+    for (const file of files) {
+        const formData = new FormData();
+
+        formData.append('files[]', file, file.name)
+        if(!localStorage.getItem("profile")) {
+            document.getElementById("no-profile-error").style.display = "block";
+            return;
+        }
+        await API.postFormData(
+            "profiles/" + localStorage.getItem("profile") + "/newentryfile",
```

Comment:
hier sollte doch das profil genutzt werden, das gerade im view geöffnet ist. Sonst fügt es den Termin ja immer zum gespeicherten Profil hinzu statt zu dem in dem ich grade bin. Das ist ja mega verwirrend.